leapt/im-bundle/2.0/config/packages/leapt_im.yaml

```diff
@@ -0,0 +1,6 @@
+leapt_im:
```

Comment:
It could be a god idea to add a link to https://github.com/leapt/im-bundle/blob/master/docs/configuration.rst

Reply:
Done